src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
Proposition on exactly *what* to move away from mongoBackend added to issue #1405